addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -76,66 +76,273 @@ function factory(dependencies) {
+                     * This is broadcast to the recipient of the needaction.
+                     */
+                    case 'mail.inbox_needaction':
+                        return this._handleNotificationNeedaction(message.payload);
+                    /**
+                     * This is when a inbox message is marked as read.
+                     * This is broadcasted to the current user for multi device synchronisation.
+                     */
+                    case 'mail.mark_as_read':
+                        return this._handleNotificationPartnerMarkAsRead(message.payload);
+                    /**
+                     * This notification is send when you "resend" a failed notification
+                     * This is broadcasted to the current user making the resend request.
+                     */
+                    case 'mail.message_notification_update':
+                        return this._handleNotificationPartnerMessageNotificationUpdate(message.payload);
+                    /**
+                     * This use only one time to display a warning that GoIP resolver is missing.
+                     */
+                    case 'mail.simple_notification':
+                        return this._handleSimpleNotification(message.payload);
+                    /**
+                     * This notification is send when the user "star" a message.
+                     * This is broadcasted to the current user for multi-device synchronisation.
+                     */
+                    case 'mail.toggle_star':
+                        return this._handleNotificationPartnerToggleStar(message.payload);
+                    /**
+                     * This notification is send when a "system" message is send. (eg: /help).
+                     * This is only send to the current use for multi-device synchronisation.
+                     */
+                    case 'mail.transient_message':
+                        return this._handleNotificationPartnerTransientMessage(message.payload);
+                    /**
+                     * This notification is send when someone is typing inside the channel.
+                     * This is broadcasted to all channel members.
+                     */
+                    case 'mail.typing_status':
+                        return this._handleNotificationChannelTypingStatus(message.payload);
+                    /**
+                     * This notification is send when a user "close" a private discution (direct message).
+                     * This is broadcasted to the current user for multi-device synchronisation.
+                     */
+                    case 'mail.unsubscribe':
+                        if (id !== this.env.messaging.currentPartner.id) {
+                            // ignore broadcast to other partners
+                            return;
+                        }
+                        return this._handleNotificationPartnerUnsubscribe(message.payload);
```

Comment:
This should be a "channel members left" notification, and the handler should take care of it according to if the leaving member is the current partner or not.